### PR TITLE
[WR212] Terraform Provider TozID Realm Federation

### PR DESCRIPTION
**Pull request by rosasck**
currently, having some issues with the new resources being able to create the SDK
while im debugging that, thought id put this up for a draft pr 
Screen Shot 2021-11-01 at 4 48 49 PM

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Configure the Terraform runtime
```
Comment (rosasck) [inline main.tf:1]: will remove before merging

File: tozny/resource_primary_realm_tozid_federation.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+			},
+		},
+	}
+}
+
+func resourcePrimaryRealmFederationCreate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	var diags diag.Diagnostics
+	var err error
+
+	toznySDK, err := MakeToznySDK(d, m)
```
Comment (rosasck) [inline tozny/resource_primary_realm_tozid_federation.go:68]: this throws error

File: tozny/resource_shadow_realm_tozid_federation.go [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+			},
+		},
+	}
+}
+
+func resourceShadowRealmFederationCreate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	var diags diag.Diagnostics
+	var err error
+
+	toznySDK, err := MakeToznySDK(d, m)
```
Comment (rosasck) [inline tozny/resource_shadow_realm_tozid_federation.go:101]: ^^

**Comment by rosasck** [comment]
Screen Shot 2021-11-01 at 7 08 47 PM

Connecting to a primary works!

**Comment by rosasck** [comment]
Screen Shot 2021-11-01 at 7 09 07 PM

heres me logging in to test

**Comment by rosasck** [comment]
Screen Shot 2021-11-01 at 7 23 01 PM

Initiating Primary Realm to federate works!

**Comment by rosasck** [comment]
Screen Shot 2021-11-01 at 7 23 40 PM

Looked for the resource id and it was correctly configured!

**Review by galxy25 — APPROVED** [review]

**Review by pirtleshell — APPROVED** [review]
super cool!

**Pull request merged**